service/src/main/java/bio/terra/pipelines/dependencies/stairway/JobMapKeys.java

```diff
@@ -11,8 +11,8 @@ public class JobMapKeys {
   public static final String PIPELINE_ID = "pipeline_id";
   public static final String STATUS_CODE = "status_code";
   public static final String RESPONSE = "response"; // result or output of the job
-  public static final String RESULT_PATH =
-      "result_path"; // path to the result API endpoint for this job; only used for asynchronous
+  // domain name for the service, used to generate the URL for the result
+  public static final String DOMAIN_NAME = "domain_name";
```

Comment:
could this be an environment/configuration variable instead? is this ever going to change job to job within the same environment?

Reply:
hmm i might be missing something, this is just the key name we're storing the domain value for a flight.  Why would we need this to be configurable?  the value being passed is coming from the ingress configuration if that's what you are referring to

Reply:
chatted with faces.  a better solution would be to grab the IngressConfiguration from a bean directly in the utils class we end up using this value in but that kind of adds other cruft to the code.  When we see more examples of us using flight params to pass information to the post flight functions then we should look into swapping over.  For one usage this is fine enough